astropy/coordinates/tests/test_sky_coord.py

```diff
@@ -902,6 +912,16 @@ def test_frame_attr_transform_inherit():
     assert c2.equinox.value == 'J1990.000'
     assert c2.obstime.value == 'J1980.000'
 
+    # The work-around for #5722
+    c  = SkyCoord(1 * u.deg, 2 * u.deg, frame='fk5')
+    c1 = SkyCoord(1 * u.deg, 2 * u.deg, frame='fk5', equinox='B1950.000')
+    c2 = c1.transform_to(c)
+    assert not c2.is_equivalent_frame(c) # counterintuitive, but documented
+    assert c2.equinox.value == 'B1950.000'
+    c3 = c1.transform_to(c, override_defaults=False)
```

Comment:
Even after dealing with this issue for many days now, I still would get confused when seeing a parameter like `override_defaults=False`.  It just has no meaning to me without looking at the docstring.  But if I saw `prioritize_destination=True`, which is opting in to change the default behavior, I would have a chance to guess/remember what that does.